resources/mediasearch-vue/init.js

```diff
@@ -8,11 +8,11 @@
 	// TODO: This should probably be handled differently
 	/* eslint-disable no-jquery/no-global-selector */
 	/* eslint-disable no-jquery/no-parse-html-literal */
-	$( '#mw-content-text' ).empty().append( $( '<div id="media-search-vue"></div>' ) );
+	$( '#mw-content-text' ).empty().append( $( '<div id="wbmi-media-search"></div>' ) );
```

Comment:
This is being presumptuous and assuming this will be the canonical media search one day :)

Reply:
The resource module is `mediasearch-vue` to keep the old code in place for comparsion, but eventually I assume that will change too.